tooling/minimize/tests/ub/tree_borrows/overlapping_write_write.rs

```diff
@@ -0,0 +1,17 @@
+extern crate intrinsics;
+use intrinsics::*;
+
+//@ compile-flags: --minimize-tree-borrows
+fn main() {
+    let parent = &mut 31; // (parent, Reserved)   
+    let x = parent as *mut i32; // (parent, Reserved) 
+    let y = unsafe { &mut *x }; // (parent, Reserved) -> (y, Reserved)
+    *y = 42;
+
+    unsafe { 
+        *x = 57; // (parent, Reserved) -> (y, Disabled)
+        print(*x); // (parent, Reserved) -> (y, Disabled)
```

Comment:
```suggestion
        assert!(*x == 57); // (parent, Reserved) -> (y, Disabled)
```
This is generally nicer than printing.